pages/common/lex.md

```diff
@@ -0,0 +1,12 @@
+# lex
+
+> Lexical analyser generator.
+> Given a lexical analyser, generates C code implementing it.
+
+- Generate analyser from Lex file:
+
+`lex {{path/to/example.l}}`
+
+- Compile a C file generated by lex:
+
+`cc {{path/to/lex.yy.c}} --output {{example}}`
```

Comment:
Should this be `example.l` ?

Reply:
It seems you didn't change it. Was that intended ?

Reply:
This is the same commit we are commenting on. :) Have a look at the latest revision.

Reply:
The latest version of your example says `cc {{path/to/lex.yy.c}} --output {{executable}}`. I believe the idea was to change `example` to `example.l`. I see that you have changed `example` to `executable`. Let me know if I am missing something.